lib/rules/consistent-output.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
         default: 'consistent',
       },
     ],
+    defaultOptions: ['consistent'],
```

Comment:
See #498 -> #503 :wink: 

I didn't remove `default` from the schemas here because eslint-doc-generator doesn't support `defaultOptions` yet (see https://github.com/bmish/eslint-doc-generator/issues/513), so removing `default`s would make the docs worse.